README.md

```diff
@@ -8,30 +8,38 @@ This tool can currently be used to update the Black Magic Debug firmware on your
 
 ## Installation
 
+Binary releases for Linux, Mac (x64/ARM) and Windows (x64/ARM) are now available with every
+[release](https://github.com/blackmagic-debug/bmputil/releases). These should work out-of-the-box,
+and do not require manual installation of Windows Driver Kit 8.0 or Rust.
+
+
+Alternately, you can install directly from [crates.io](https://crates.io/crates/bmputil) with cargo.
+
 First install Rust on your computer. Follow the instructions on the [Rust Lang website](https://www.rust-lang.org/tools/install).
 
-Now you can build and install the tool from source.
+Then, install bmputil using `cargo install bmputil`
+
+bmputil on Windows will attempt to automatically setup driver installation on first run.
+This is extra experimental, and will require administrator access on the first run.
+
+## Building from source
+Alternatively, you can build and install the tool from source. This assumes that you have Rust (and
+git, etc) installed already.
 ```
 git clone https://github.com/blackmagic-debug/bmputil.git
 cd bmputil
 cargo install --path .
 ```
 
-If you are working on patches or contributions to the tool, you can obviously use `cargo build` and `cargo run [params]` as needed.
-
-**Note:** This tool is not yet listed on crates.io. So unfortunately you can't install it using cargo directly yet.
-**Note:** We don't currently have pre-built binaries/installers available. But they are planned.
-
-## Windows
-
-Installation is done with the above instructions, with one additional prerequisite. You will need to install the [Windows
-Driver Kit 8.0 redistributable components](https://go.microsoft.com/fwlink/p/?LinkID=253170) (link from
-[this](https://learn.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads) page). If you are cross
-compiling to Windows, you will need to set the `WDK_DIR` environment variable to the path of the extracted WDK
-redistributable components.
+If you are working on patches or contributions to the tool, you can obviously use `cargo build` and
+`cargo run [params]` as needed.
 
-bmputil on Windows will also attempt to automatically setup driver installation on first run. This is extra
-experimental, and will require administrator access on the first run.
+### Windows
+For building bmputil locally for a windows platform (either on windows or cross-compiling), you will
+need to install the [Windows Driver Kit 8.0 redistributable components](https://go.microsoft.com/fwlink/p/?LinkID=253170)
+(link from [this](https://learn.microsoft.com/en-us/windows-hardware/drivers/other-wdk-downloads) page).
+If you are cross compiling to Windows, you will need to set the `WDK_DIR` environment variable to
+the path of the extracted WDK redistributable components.
 
```

Comment:
Our bad that there's an extra new line here, please clean it up with the other lint fixes.